deployment/argo-workflows/eks/workflow-treemodel-scene-id.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: workflow-
+spec:
+  entrypoint: workflow-template
+  parallelism: 30
+  arguments:
+    parameters:
+    - name: activator-input
+      value: |
+        ["f140603t01p00r18_sc01"]
+
+  templates:
+  - name: workflow-template
+    dag:
+      tasks:
+      - name: activator
+        template: activator-template
+        arguments:
+          parameters:
+          - name: activator-input
+            value: "{{item}}"
+        withParam: "{{workflow.parameters.activator-input}}"
+
+  - name: activator-template
+    inputs:
+      parameters:
+      - name: activator-input
+    retryStrategy:
+      limit: "10"
+      expression: "asInt(lastRetry.exitCode) == -42"
+      # https://argoproj.github.io/argo-workflows/fields/#backoff
+      backoff:
+        duration: "10m" # Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
+        factor: "2"
+        maxDuration: "1d"
+    container:
+      image: quay.io/jmcclain/activator:latest
```

Comment:
What's the nature of this container?  Is it general enough to land in an Azavea-owned repository?

Reply:
Yes, as discussed above, the plan is to push it to ECR after PR approval.

Reply:
:heavy_check_mark: